packages/app-store/facetime/components/InstallAppButton.tsx

```diff
@@ -0,0 +1,18 @@
+import type { InstallAppButtonProps } from "@calcom/app-store/types";
```

Comment:
We don't need to add this Component separately now. It's automatically handled. Can you try deleting this file and see if it works still.

Reply:
Removed this file and _metadata.ts. Those files are not required now. New apps created using cli don't have them.
There is an issue in which we need to cleanup old apps https://github.com/calcom/cal.com/issues/6714